src/main/java/com/puppycrawl/tools/checkstyle/site/ModuleJavadocParsingUtil.java

```diff
@@ -172,7 +172,7 @@ public static int getPropertySectionStartIndex(DetailNode moduleJavadoc,
 
         if (somePropertyModuleNode.isPresent()) {
             propertySectionStartIndex = JavadocMetadataScraper.getParentIndexOf(
-                somePropertyModuleNode.get());
+                    somePropertyModuleNode.orElseThrow());
```

Comment:
again format is broken but this no new issue.

```suggestion
                somePropertyModuleNode.orElseThrow());
```